generators/client/templates/angular/src/main/webapp/app/shared/_shared-common.module.ts

```diff
@@ -6,6 +6,10 @@ import { TranslateService } from 'ng2-translate';
 <%_ } _%>
 import { AlertService } from 'ng-jhipster';
 
+<%_ if (websocket === 'spring-websocket') { _%>
+import { WindowRef } from './tracker/window.service';
```

Comment:
I dont want to add it in lib because majority of people wont even use websockets. So it is unnecessary overhead. Let our lib be simple and we add what is used across. IMO